README.md

```diff
@@ -9,259 +16,134 @@
+                ],
+                "Resource": [
+                    "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/*",
+                    "arn:aws:iam::<ACCOUNT_ID>:role/*"
+                ]
+            }
+        ]
     }
-    ocm = {
-      version = "0.0.3"
-      source  = "terraform-redhat/ocm"
-    }
-  }
-}
-
-locals {
-  sts_roles = {
-      role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.account_role_prefix}-Installer-Role",
-      support_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.account_role_prefix}-Support-Role",
-      instance_iam_roles = {
-        master_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.account_role_prefix}-ControlPlane-Role",
-        worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.account_role_prefix}-Worker-Role"
-      },
-      operator_role_prefix = var.operator_role_prefix,
-  }
-}
-
-data "aws_caller_identity" "current" {}
+    ```
 
-resource "ocm_cluster_rosa_classic" "rosa_sts_cluster" {
-  name           = var.cluster_name
-  cloud_region   = var.region
-  aws_account_id     = data.aws_caller_identity.current.account_id
-  availability_zones = [var.zone]
-  properties = {
-    rosa_creator_arn = data.aws_caller_identity.current.arn
-  }
-  sts = local.sts_roles
-  # disable_waiting_in_destroy = false
-  # destroy_timeout in minutes
-  destroy_timeout = 60
-}
+## Terraform examples
 
-resource "ocm_cluster_wait" "rosa_cluster" {
-  cluster = ocm_cluster_rosa_classic.rosa_sts_cluster.id
-  # timeout in minutes
-  timeout = 60
-}
+The following examples show complete, usable Terraform examples. You must configure the examples with your own credentials. After the files are configured, run the following commands to test these examples:
 
-data "ocm_rosa_operator_roles" "operator_roles" {
-  operator_role_prefix = var.operator_role_prefix
-  account_role_prefix = var.account_role_prefix
-}
+1. `terraform init` - This command uses all the necessary providers for your project.
+1. `terraform plan -out <name>.tfplan` - **Optional**: This command creates the Terraform plan file that you use to create your infrastructure. This command serves as a "dry run" to ensure that the Terraform will execute properly.
+1. `terraform apply "<name>.tfplan"` - This command executes the Terraform process and creates the needed resources to produce your desired infrastructure.
 
-module operator_roles {
-  source = "terraform-redhat/rosa-sts/aws"
-  version = "0.0.4"
+The example Terraform files are all considered in development:
 
-  create_operator_roles = true
-  create_oidc_provider = true
-  create_account_roles = false
+1. **Required**: [Account Roles terraform](https://github.com/terraform-redhat/terraform-provider-ocm/blob/a42779d6b6712f4dde358344f44b782e4dfcd120/examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md)
```

Comment:
You already added a link to this in the pre-requisites, so maybe this is not needed here, what do you say?

Reply:
This is a little different - I am using these as examples in the README. If I am including the clusters, I feel like I should also include the account roles, BUT since I am writing that PR in parallel I'll remove this item from here and make it a pre-req within the cluster creation pages.